articles/azure-monitor/app/tutorial-asp-net-core.md

```diff
@@ -0,0 +1,525 @@
+
+    ```csharp
+    builder.Services.AddApplicationInsightsTelemetry();
+    ```
+
+    ![A code window displays with the preceding code snippet highlighted.](./media/asp-net-core/enable_server_side_telemetry.png "Enable server-side telemetry")
+
+    > [!TIP]
+    > Learn more about [configuration options in ASP.NET Core](https://docs.microsoft.com/aspnet/core/fundamentals/configuration/?view=aspnetcore-6.0).
+
+## Enable client-side telemetry for web applications
+
+The preceding steps are enough to help you start collecting server-side telemetry. If your application has client-side components, follow the next steps to start collecting [usage telemetry](./usage-overview.md).
+
+1. In Visual Studio Solution explorer, locate and open `\Views\_ViewImports.cshtml`. Add the following code at the end of the existing file.
+
+    ```cshtml
+    @inject Microsoft.ApplicationInsights.AspNetCore.JavaScriptSnippet JavaScriptSnippet
+    ```
+
+    ![The _ViewImports.cshtml file displays with the preceding line of code highlighted.](./media/asp-net-core/view_imports_injection.png "JavaScriptSnippet injection")
+
+2. To properly enable client-side monitoring for your application, the JavaScript snippet must appear in the `<head>` section of each page of your application that you want to monitor. In Visual Studio Solution Explorer, locate and open  `\Views\Shared\_Layout.cshtml`, insert the following code immediately preceding the closing `<\head>` tag.
+
+    ```cshtml
+    @Html.Raw(JavaScriptSnippet.FullScript)
+    ```
+
+    ![The _Layout.cshtml file displays with the preceding line of code highlighted within the head section of the page.](./media/asp-net-core/layout_head_code.png "The head section of _Layout.cshtml")
+
+    > [!TIP]
+    > As an alternative to using the `FullScript`, the `ScriptBody` is available starting in Application Insights SDK for ASP.NET Core version 2.14. Use `ScriptBody` if you need to control the `<script>` tag to set a Content Security Policy:
+
+    ```cshtml
+    <script> // apply custom changes to this script tag.
+        @Html.Raw(JavaScriptSnippet.ScriptBody)
+    </script>
+    ```
+
+> [!NOTE]
+> JavaScript injection provides a default configuration experience. If you require [configuration](https://docs.microsoft.com/azure/azure-monitor/app/javascript#configuration) beyond setting the connection string, you are required to remove auto-injection as described above and manually add the [JavaScript SDK](https://docs.microsoft.com/azure/azure-monitor/app/javascript#adding-the-javascript-sdk).
+
+## Enable monitoring of database queries
+
+When investigating causes for performance degredation, it is important to include insights into database calls. Enable monitoring through configuration of the [dependency module](https://docs.microsoft.com/azure/azure-monitor/app/asp-net-dependencies).
```

Comment:
dependency monitoring including SQL is already enabled even without this step. This step is only relevant to capture the full SQL query text itself. So the wording should reflect it. (along with a warning that SQL text might contain passwords etc, so be careful enabling it..)

Reply:
Fixed.